test/integration/short/client-each-row-tests.js

```diff
@@ -534,37 +534,10 @@ describe('Client', function () {
       });
     });
     it('should retrieve large result sets in parallel', function (done) {
-      var client = newInstance({ queryOptions: {
-        consistency: types.consistencies.quorum,
-        fetchSize: 20000
-      }});
-      client.on('log', function (level, className, message, furtherInfo) {
-        if (level !== 'warning' && level !== 'error') {
-          return;
-        }
-        console.error(level, className, message, furtherInfo);
-      });
-      var query = util.format('SELECT * FROM %s LIMIT 2000', table);
-      utils.series([
-        client.connect.bind(client),
-        helper.toTask(insertTestData, null, client, table, 2000),
-        function selectData(seriesNext) {
-          utils.timesLimit(400, 6, function (n, timesNext) {
-            var counter = 0;
-            client.eachRow(query, [], { prepare: true }, function (n, row) {
-              assert.ok(row);
-              counter++;
-            }, function (err, result) {
-              assert.ifError(err);
-              assert.strictEqual(result.rowLength, counter);
-              timesNext();
-            });
-          }, seriesNext);
-        }
-      ], function (err) {
-        client.shutdown();
-        done(err);
-      });
+      insertSelectTest(table, 50000, 50, 50000, { prepare: true}, done);
+    });
+    it('should query multiple times in parallel with query tracing enabled', function (done) {
+      insertSelectTest(table, 50000, 10000, 10, { prepare: true, traceQuery: true }, done);
```

Comment:
When sending multiple requests to read, it's likely that the response chunk received may contain an incomplete frame (the last one might be sliced in multiple chunks).

I'm trying to have incomplete frames with streaming enabled as a way to reproduce the original issue, but this may not occur on all runs...